cgi-bin/LJ/CleanHTML.pm

```diff
@@ -532,13 +532,6 @@ TOKEN:
                 $ljcut_div = 0;
             }
 
-            # no cut URL, record the anchor, but then fall through
-            if ( 0 && $ljcut_div && !$cut ) {
```

Comment:
Over 14 years, I wonder just how much CPU we spent on the Perl interpreter reading this code and deciding to throw it away. And I wonder if, at any point, it bitflipped and ran it...